gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineNullAway.java

```diff
@@ -36,6 +38,14 @@ public final class BaselineNullAway implements Plugin<Project> {
     /** We may add a gradle extension in a future release allowing custom additional packages. */
     private static final ImmutableSet<String> DEFAULT_ANNOTATED_PACKAGES = ImmutableSet.of("com.palantir");
 
+    private static final ImmutableList<String> NON_15_DEPS = ImmutableList.of(
+            "com.uber.nullaway:nullaway:0.10.1",
+            // align 'org.checkerframework' dependency versions on
+            // the current latest version.
+            "org.checkerframework:dataflow-errorprone:3.25.0",
+            "org.checkerframework:dataflow-nullaway:3.25.0",
+            "org.checkerframework:checker-qual:3.25.0");
```

Comment:
We should probably also move nullaway and checkerframework to this section in versions.props to prevent excavator updating it as direct dependency.

https://github.com/palantir/gradle-baseline/blob/c1552fa1c5eb551aa6b4316aa07521251c17077e/versions.props#L42-L48

Reply:
I could go either way, we do want to upgrade it eventually, and our test coverage will prevent version-bumps from going through (jdk-15 nullaway integration test won't work!)

Reply:
Ah yeah, just saw that you added regression tests in https://github.com/palantir/gradle-baseline/pull/2389

Reply:
Baseline-java-versions is super helpful for this style of testing :-]